dockerfiles/splash/download-qt-installer.sh

```diff
@@ -2,6 +2,6 @@
 
 # XXX: if qt version is changed, Dockerfile should be updated,
 # as well as qt-installer-noninteractive.qs script.
-URL="http://download.qt.io/official_releases/qt/5.13/5.13.1/qt-opensource-linux-x64-5.13.1.run"
+URL="http://download.qt.io/official_releases/qt/5.14/5.14.1/qt-opensource-linux-x64-5.14.1.run"
```

Comment:
Why not 5.14.2?

Reply:
https://github.com/qtwebkit/qtwebkit/releases/tag/qtwebkit-5.212.0-alpha4 said to use 5.14.1, I'm sure the patch version wouldn't make too much difference but wanted to follow the instructions.

Reply:
makes sense!